Tests/Marketplace/upload_packs.py

```diff
@@ -1088,20 +1063,23 @@ def main():
                                                                         is_bucket_upload_flow, ci_branch)
 
     # detect packs to upload
-    pack_names = get_packs_names(target_packs, previous_commit_hash)  # list of the pack's ids to upload
+    pack_names_to_upload = get_packs_names(packs_to_upload)
     extract_packs_artifacts(packs_artifacts_path, extract_destination_path)
-    # this is a list of all packs from `content_packs.zip`
-    content_packs = [Pack(pack_name, os.path.join(extract_destination_path, pack_name))
-                     for pack_name in os.listdir(extract_destination_path)]
+    # list of all packs from `content_packs.zip` given from create artifacts
+    all_content_packs = [Pack(pack_name, os.path.join(extract_destination_path, pack_name),
+                              is_modified=pack_name in pack_names_to_upload)
+                         for pack_name in os.listdir(extract_destination_path) if pack_name != 'index']
```

Comment:
not sure I understood, why will we have at this stage a file called `index` (without extension)?

Reply:
added `index` to IGNORED_FILES